ecosystem/typescript/sdk/examples/javascript/bcs_transaction.js

```diff
@@ -0,0 +1,68 @@
+const aptos = require('aptos');
+
+const NODE_URL = process.env.APTOS_NODE_URL || 'https://fullnode.devnet.aptoslabs.com';
+const FAUCET_URL = process.env.APTOS_FAUCET_URL || 'https://faucet.devnet.aptoslabs.com';
+
+const { AptosClient, AptosAccount, FaucetClient, BCS, TxnBuilderTypes } = aptos;
+
+const {
+  AccountAddress,
+  TypeTagStruct,
+  ScriptFunction,
+  StructTag,
+  TransactionPayloadScriptFunction,
+  RawTransaction,
+  ChainId,
+} = TxnBuilderTypes;
+
+(async () => {
+  const client = new AptosClient(NODE_URL);
+  const faucetClient = new FaucetClient(NODE_URL, FAUCET_URL, null);
```

Comment:
I think if these are examples that people are going to use, it needs comments explaining what is going on.  I see it as someone with little knowledge should be able to read it and learn about it.